distribution/common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -0,0 +1,129 @@
+#    limitations under the License.
+#
+################################################################################
+
+# In order to switch to Log4j2 and be able to take advantage of the newer appenders,
+# such as the Failover appender, uncomment the following line and remove all the
+# configuration beneath it.
+#
+# This change can be made permanent (or done in another way, depending upon
+# the implementation choices made) when we have upgraded to Karaf 4.1.0
+# https://codice.atlassian.net/browse/DDF-2132
+#
+# org.ops4j.pax.logging.log4j2.config.file = ${karaf.etc}/log4j2.config.xml
+
+# Root logger
+log4j.rootLogger=INFO, out, osgi:*
+log4j.throwableRenderer=org.apache.log4j.OsgiThrowableRenderer
+
+# Security logger
+log4j.logger.securityLogger=INFO, security, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.securityLogger=false
+
+# Ingest Logger
+log4j.logger.ingestLogger = INFO, ingestError, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.ingestLogger=false
+
+# Solr Logger
+log4j.logger.org.apache.solr = INFO, solr, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.org.apache.solr = false
+log4j.logger.org.apache.lucene = INFO, solr, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.org.apache.lucene = false
+
+# Artemis Logger
+log4j.logger.org.apache.activemq.artemis = INFO, artemis, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.org.apache.activemq.artemis=false
+
+# LogAuditLoginModule logger
+log4j.logger.org.apache.karaf.jaas.modules.audit.LogAuditLoginModule=INFO, security, syslog, osgi:org.codice.ddf.platform.logging.LoggingService
+log4j.additivity.org.apache.karaf.jaas.modules.audit=false
+
+# Syslog appender
+log4j.appender.syslog=org.apache.log4j.net.SyslogAppender
+log4j.appender.syslog.layout=org.apache.log4j.PatternLayout
+log4j.appender.syslog.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
+log4j.appender.syslog.facility=AUTH
+# this should be set to loghost generally when a syslog host is configured
+log4j.appender.syslog.syslogHost=localhost
+
+# CONSOLE appender not used by default
+log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+log4j.appender.stdout.layout.ConversionPattern=%d{ABSOLUTE} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
+
+# File appender
+log4j.appender.out=org.apache.log4j.RollingFileAppender
+log4j.appender.out.layout=org.apache.log4j.PatternLayout
+log4j.appender.out.layout.ConversionPattern=%d{ISO8601} | %-5.5p | %-16.16t | %-32.32c{1} | %-32.32C %4L | %X{bundle.id} - %X{bundle.name} - %X{bundle.version} | %m%n
+log4j.appender.out.file=${karaf.data}/log/alliance.log
```

Comment:
Using a version property here (i.e. `${branding-lowercase}.log` instead of `alliance.log`) causes the log file to be named `.log`, so `alliance.log` is used here instead.